### PR TITLE
Added update instructions for Prefix packages and Portage

### PR DESCRIPTION
**Pull request by peterstol**
Added  instructions to update portage and packages to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,3 +64,40 @@ Sync the overlay
...
+emerge --sync
+emerge
+```
+If you run into problems, usually a newer ebuild is not suited to build in a prefix environment.
+Try to mask latest versions:
+
+Create a mask file if not existing and mask newer versions from thin provisioning tools greater or equal to 0.7.6:
+```
+touch /path/to/prefix/etc/portage/package.mask
+echo ">=sys-block/thin-provisioning-tools-0.7.6" >> /path/to/prefix/etc/portage/package.mask
```
Comment (boegel) [inline README.md:81]: Use `$EPREFIX`, so it's consistent with the above? Also, the `touch` is not needed?

```
export EPREFIX=/path/to/prefix
echo ">=sys-block/thin-provisioning-tools-0.7.6" >> $EPREFIX/etc/portage/package.mask

File: README.md [hunk 2]
```diff
@@ -64,3 +64,40 @@ Sync the overlay
...
+```
+
+#### Portage
+Updating Portage requires the kernel source which corresponds to your running kernel on the host. Emerge will detect it in /usr/src/linux.
```
Comment (boegel) [inline README.md:85]:
```suggestion
Updating Portage requires the kernel source which corresponds to your running kernel on the host. Emerge will detect it in `/usr/src/linux`.
```

File: README.md [hunk 3]
```diff
@@ -64,3 +64,40 @@ Sync the overlay
...
+
+Check your running kernel version with:
+```
+cat /proc/version
+Linux version 4.20.0-1.el7.elrepo.x86_64 (mockbuild@Build64R7) 
+```
+
+On a Centos 7 host kernel sources are installed in /usr/src/kernels. Link /usr/src/linux to the appropiate kernel source after installation. Example an elrepo kernel:
```
Comment (boegel) [inline README.md:93]:
```suggestion
On a Centos 7 host kernel sources are installed in `/usr/src/kernels`. Link `/usr/src/linux` to the appropiate kernel source after installation. Example for an `elrepo` kernel:
```

File: README.md [hunk 4]
```diff
@@ -64,3 +64,40 @@ Sync the overlay
...
+```
+rpm -ivh kernel-ml-devel-4.20.0-1.el7.elrepo.x86_64.rpm
+cd /usr/src ; ln -s kernels/4.20.0-1.el7.elrepo.x86_64 linux
+```
+
+When ready update portage from the prefix:
```
Comment (boegel) [inline README.md:99]:
```suggestion
 When ready update Portage from the Prefix environment:
```

File: README.md (hunk 1 above)
Comment (peterstol) [reply README.md]: agreed

**Review by boegel — APPROVED** [review]

**Pull request merged**